ecmascript/minifier/tests/compress/fixture/issues/2262/1/input.js

```diff
@@ -0,0 +1,20 @@
+(() => {
+    "use strict";
+
+    var commonjsGlobal = globalThis
+
+    function createEventEmitter(value) {
+    }
+
+    var index = somethingGlobal;
+
+    const esm = index;
```

Comment:
This indirection is important to the bug. `const esm = somethingGlobal `wouldn't reproduce it either.